packages/lime-system/files/usr/lib/lua/lime/wireless.lua

```diff
@@ -86,7 +86,7 @@ function wireless.configure()
 
 		if specRadio then
 			modes = specRadio["modes"]
-			options = specRadio
+			utils.tableConcat(options, specRadio)
```

Comment:
What happens if an option is present both in general and in specific section?

Reply:
as it's added at the end the "default" options are overwritten with specific ones.

Reply:
Better add a documentation comment there explaining this as this behavior is implicit

Reply:
Apart from the missing documentation you're okay with this PR?

Reply:
Yeah for me it is ok.
I am wondering if similar behavior is something one would have on specific network interfaces too, I was considering this also because of homogeneity between the two specific behaviors